osimpipeline/vital_tasks.py

```diff
@@ -653,22 +701,25 @@ def __init__(self, tool_folder, trial, cycle=None):
                 'Kinematics_dudt.sto', 'Kinematics_q.sto', 'Kinematics_u.sto',
                 'pErr.sto', 'states.sto']:
             self.targets += ['%s/results/%s_%s_%s_%s' % (
-                self.path, self.study.name, trial.id, tool_folder, rra_output)]
+                self.path, self.study.name, cycle.id if cycle else trial.id,
```

Comment:
I think you should leave this as-is for now, but the if statement motivates the use of `tricycle`, as this could be `tricycle.id`.